### PR TITLE
feat: 🎸 add placeholder to Input.Number

### PR DESCRIPTION
**Pull request by jonasjeeliasson**
We need a placeholder for the Input.Number.

### CODE REVIEW

File: src/Molecules/Input/Number/Number.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import NormalizedElements from '../../../common/NormalizedElements';
 import { getStringAsNumber, getNumberAsString } from './utils';
 import { isNumber, isString, isUndefined, assert } from '../../../common/utils';
 import adjustValue from './adjustValue';
+import { placeholderNormalizaion } from '../Text/Text';
```
Comment (jonasjeeliasson) [inline src/Molecules/Input/Number/Number.tsx]: I reused the placeholderNormalization from Text (the same way is used in TextArea)

**Comment by jonasjeeliasson** [comment]
Hmm got some conflicts, will have to check them tomorrow.

**Comment by jonasjeeliasson** [comment]
> Hmm got some conflicts, will have to check them tomorrow.

**Comment by mjmonline** [comment]
How does the use case look? Also if there is a valid use case then include a story of it

**Comment by jonasjeeliasson** [comment]
Hi,

The use case is that we need placeholder saying "Belopp". For all except
one of our number inputs, we want this placeholder.

Do you mean I should include a screenshot here displaying its actual use
case or that I should change the story in storybook with a real use case
instead of the generic one with the text placeholder which I've already
added?

**Comment by shmidt-i** [comment]
@jonasjeeliasson sry for late answer, could you please change placeholder to only appear when there is `noSteppers`? that's the only thing I didn't like couple of days ago, but then totally forgot about this ;)

**Comment by jonasjeeliasson** [comment]
@shmidt-i Now the placeholder won't be visible with steppers. Added multiple cases in the storybook to show of how it looks with different addons and such.

**Pull request merged**